packages/std/src/os/kernel.rs

```diff
@@ -102,6 +106,8 @@ pub enum QueryMsg {
     ChannelInfo { chain: String },
     #[returns(Vec<::cosmwasm_std::Coin>)]
     Recoveries { addr: Addr },
+    #[returns(String)]
```

Comment:
This could return a custom struct instead like `ChainNameResponse`, to be more consistent with the return values of `Version` and `ChannelInfo`. Maybe we should apply that change to all the queries that aren't returning a custom struct.